code/index.html

```diff
@@ -1,30 +1,254 @@
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+    <title>Dream cakes 🎂</title>
+    <link rel="stylesheet" href="style.css">
+  </head><body>
 
+  <header>
+    <div class="overlay">
+      <h1>Dream Cakes</h1>
+      <p class="tagline">Create your own dream cake<br></p>
+      <button class="btn">Contact us</button>
+    </div>
+   
+        
+    <div class="video1">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo.mp4" type="video/mp4">
+      </video>
+    </div>
+
+    <div class="video2">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo2.mp4" type="video/mp4">
+      </video>
+    </div>
+
+    <div class="video3">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo3.mp4" type="video/mp4">
+      </video>
+    </div>
+
+  </header>
+
+    
     <!-- Signup form -->
+  <section class="content">
+    <form 
+    class="form"
+    action="http://httpbin.org/anything"
+    method="POST"
+    >
+
+    <h2>Get your dream cake</h2>
+    <p>Choose a cake style from the list to the
+       right and we will send you an offer</p>
+    
+    <div class="form-group">
+      <label for="firstName">First name*</label>
+      <input
+      id="firstName"
+      type="text"
+      placeholder="Type your first name"
+      name="firstNameValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="lasttName">Last name*</label>
+      <input
+      id="lasttName"
+      type="text"
+      placeholder="Type your last name"
+      name="lastNameValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="email">Email*</label>
+      <input
+      id="email"
+      type="email"
+      placeholder="Type your e-mail"
+      name="emailValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="message">What cake style do you want?</label><br>
+      <textarea
+      id="message"
+      cols="30"
+      rows="2"
+      maxlength="100"
+      name="messageValue"
+      placeholder="Max. 100 characters"
+      required
+       ></textarea>
+    </div>
+
+      <label>How many pieces of cake?</label>
+
+    <div class="form-group">
+      <input 
+      type="radio" 
+      id="8" 
+      name="pieces" 
+      value="8"
+      <label for="8"> 8</label><br>
+
+
+      <input 
+      type="radio" 
+      id="12" 
+      name="pieces" 
+      value="12">
+      <label for="12">12</label><br>
+      
+
+      <input 
+      type="radio" 
+      id="16" 
+      name="pieces" 
+      value="16">
+      <label for="16">16</label><br>
+
+
+      <input 
+      type="radio" 
+      id="20" 
+      name="pieces" 
+      value="20">
+      <label for="20">20</label><br>
+    </div>
+
+      <label>Any food preferences?</label>
```

Comment:
If `label` is not connected to any `input` element, you can use `p` instead, it's better that way